sass/components/_modal.scss

```diff
@@ -338,13 +338,14 @@
 
         .modal-image__content {
             background: #F3F3F3;
-            max-height: 91.5vh;
-            max-width: 96vw;
+            height: 90vh;
+            width: 90vw;
             overflow-x: hidden;
             overflow-y: visible;
```

Comment:
I believe the horizontal scrolling bar issue has something to do with these overflows. I've tried different variations of `overflow-x` and `overflow-y` to include adding/modifying child styling classes, but no luck.